code-challenges.js

```diff
@@ -11,24 +11,64 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
+//Create a function that determines if a temperature is ...
+const boilingPoint = (temperature) => {
+    // determines if temp is at boiling point
+    if (temperature === 212){
+        return `${temperature} is at boiling point`
+    }
+    // determines if temp is above boiling point
+    else if (temperature > 212){
+        return `${temperature} is above boiling point`
+    
+    // determines if temp is below boiling point}
+    } 
+    else if (temperature < 212){
+        return `${temperature} is below boiling point`
+    }
+}
+console.log(boilingPoint(temp1));
+console.log(boilingPoint(temp2));
+console.log(boilingPoint(temp3));
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
-
-
-
+// Create a function that multiplies each number in the array by 5
+const multiplyBy5Loop = (array) => {
+// create new array to push in new values
+    let arrayMult5 = []
+// iterate through array
+    for (let i = 0; i < array.length; i++){
+// multiply values of array by 5 and push into new array
+       arrayMult5.push((array[i] *5))
+       }
+// return new array full of values
+    return arrayMult5
+}
+console.log(multiplyBy5Loop(myNumbers1));
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
+//Create a function that multiplies each number in the array
+const multiplyBy5Map = (array) => {
+//must iterate through each value in array
+    for (let i = 0; i < array.length; i++){
```

Comment:
Interesting you decided to use a for loop here, then decide to use a .map inside of it. I'm not sure exactly what the for loop is doing that the map isn't.